examples/simple/main.py

```diff
@@ -2,11 +2,14 @@
 import fairing
 import tensorflow as tf
 
-class SimpleModel():
-    def train(self):
-        hostname = tf.constant(os.environ['HOSTNAME'])
-        sess = tf.Session()
-        print('Hostname: ', sess.run(hostname).decode('utf-8'))
+fairing.config.set_builder('docker', push=False)
+fairing.config.set_deployer('job')
 
-fairing.config.set_model(SimpleModel())
-fairing.config.run()
\ No newline at end of file
+def train():
+    hostname = tf.constant(os.environ['HOSTNAME'])
+    sess = tf.Session()
+    print('Hostname: ', sess.run(hostname).decode('utf-8'))
+
+if __name__ == '__main__':
```

Comment:
can you also add an example in a notebook?

Reply:
Will do

Reply:
This won't work in the notebook yet - we'll need #69

Reply:
ok. #74 can handle this use case too.